### ISSUE TITLE
Add support for learning rate optimization for TF and Pytorch

### ISSUE THREAD

**Issue by brandon-m-booth** [emotive-computing/common-models-redesign#32]
These libraries have learning rate discovery built it, but we need support
